iio/iio_app/iio_app.c

```diff
@@ -36,6 +36,9 @@
  * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 *******************************************************************************/
+
+#ifdef IIO_SUPPORT
```

Comment:
another change that has to do with more than ad7134.

Reply:
it is a commit unrelated to ad7134.

Reply:
I was a bit confused about the PR title.

Reply:
My mistake, I changed it :)